cloud/azure/bin/ses-to-keyvault

```diff
@@ -2,27 +2,31 @@
 
 # TODO: figure out where in command flow to export PROJECT_BASE_DIR to
 # make it available in this file
-PROJECT_BASE_DIR="$(realpath "$(dirname "${BASH_SOURCE}")/../../../")"
+readonly PROJECT_BASE_DIR="$(realpath "$(dirname "${BASH_SOURCE}")/../../../")"
 source "${PROJECT_BASE_DIR}/cloud/azure/bin/lib.sh"
 source "${PROJECT_BASE_DIR}/cloud/aws/bin/lib.sh"
 
-AWS_SECRET_ACCESS_TOKEN_NAME="aws-secret-access-token"
-AWS_ACCESS_KEY_ID_NAME="aws-access-key-id"
+readonly AWS_SECRET_ACCESS_TOKEN_NAME="aws-secret-access-token"
+readonly AWS_ACCESS_KEY_ID_NAME="aws-access-key-id"
 
 # DOC: Get SES secrets from AWS CLI and add to Azure Key Vault
 
-while getopts g:v:l:u: flag
+while getopts v:u: flag
 do
   case "${flag}" in
-    v) vault_name=${OPTARG};;
-    u) username=${OPTARG};;
+    v) vault_name="${OPTARG}";;
+    u) username="${OPTARG}";;
+    *) out::error "Unexpected getopts value ${flag}";;
   esac
 done
 
-if || [[ ! "${vault_name}" ]] \
+if [[ ! "${vault_name}" ]] \
    || [[ ! "${username}" ]]; then
   out::error "arguments -v and -u must be provided"
   exit 1
+else
```

Comment:
I think the else wrapper is unnecessary here. Typically styleguides wouldn't have them when the if is terminal.

Likewise the readonly's can go right after the while loop which is what sets them.